### PR TITLE
Update set-password-to-never-expire.md

### PR DESCRIPTION
**Pull request by ziggy-msft**
Updated navigation guidance to find the setting and the correct portal

### CODE REVIEW

File: microsoft-365/admin/add-users/set-password-to-never-expire.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ description: "Learn how to set some individual user passwords to never expire, u
 
 ## Set the password expiration policy for your organization
 
-1. In the admin center, go to the **Settings** \> <a href="https://go.microsoft.com/fwlink/p/?linkid=2072756" target="_blank">Settings</a> page.
+1. In the M365 Admin Center, go to the **Settings**, **Org settings** \> <a href="https://go.microsoft.com/fwlink/p/?linkid=2072756" target="_blank">Settings</a> page.
```
Comment (cmcatee-MSFT) [inline microsoft-365/admin/add-users/set-password-to-never-expire.md:30]: Please replace step 1 with the following:

`1. In the Microsoft 365 admin center, go to the **Settings** \> <a href="https://go.microsoft.com/fwlink/p/?linkid=2072756" target="_blank">Org settings</a> page.`

File: microsoft-365/admin/add-users/set-password-to-never-expire.md [hunk 2]
```diff
@@ -27,7 +27,7 @@ description: "Learn how to set some individual user passwords to never expire, u
...
 2. At the top of the Settings page select **Security & Privacy**.
```
Comment (cmcatee-MSFT) [inline microsoft-365/admin/add-users/set-password-to-never-expire.md:31]: Please replace with the following:

`2. Select the **Security & privacy** tab.`

File: microsoft-365/admin/add-users/set-password-to-never-expire.md [hunk 3]
```diff
@@ -27,7 +27,7 @@ description: "Learn how to set some individual user passwords to never expire, u
...
 3. Select **Password expiration policy**. 
 4. If passwords are set to never expire, click the check box next to **Set user passwords to expire after a number of days**. You'll get the option to specify the number of days until passwords expire.
```
Comment (cmcatee-MSFT) [inline microsoft-365/admin/add-users/set-password-to-never-expire.md:33]: After step 4, please add:

`5. Select **Save**.`

**Review by cmcatee-MSFT — CHANGES_REQUESTED** [review]
Please make the changes as noted.

**Comment by yogkumgit** [comment]
@ziggynemeth

**Comment by cmcatee-MSFT** [comment]
The procedure in question has been removed from the article. No changes needed.

**Pull request closed without merging**